backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -923,37 +946,35 @@ public DiagnosticReport convertToDiagnosticReport(TestEvent testEvent, Date curr
       dateIssued = ZonedDateTime.ofInstant(currentDate.toInstant(), ZoneOffset.UTC);
 
     return convertToDiagnosticReport(
-        status, code, Objects.toString(testEvent.getInternalId(), ""), dateTested, dateIssued);
+        ConvertToDiagnosticReportProps.builder()
+            .status(status)
+            .code(code)
+            .id(Objects.toString(testEvent.getInternalId(), ""))
+            .effective(dateTested)
+            .issued(dateIssued)
+            .build());
   }
 
-  /**
-   * @param status Diagnostic report status
```

Comment:
since we refactored this to use the builder pattern, moved these param descriptions over to the config object